modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -253,37 +253,33 @@ public function filter_salt( $salt, $scheme ) {
 	}
 
 	/**
-	 * Enqueue the scripts for the WordPress.com block editor integration.
+	 * Enqueues the WordPress.com block editor integration assets for the editor.
 	 */
-	public function enqueue_scripts() {
+	public function enqueue_block_editor_assets() {
 		$debug   = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG;
 		$version = gmdate( 'Ymd' );
 
-		$src_common = $debug
-			? '//widgets.wp.com/wpcom-block-editor/common.js?minify=false'
-			: '//widgets.wp.com/wpcom-block-editor/common.min.js';
-
 		wp_enqueue_script(
-			'wpcom-block-editor-common',
-			$src_common,
+			'wpcom-block-editor-default-script',
+			$debug
+				? '//widgets.wp.com/wpcom-block-editor/default.js?minify=false'
+				: '//widgets.wp.com/wpcom-block-editor/default.min.js',
 			array(
 				'jquery',
 				'lodash',
-				'wp-blocks',
 				'wp-compose',
 				'wp-data',
-				'wp-dom-ready',
 				'wp-editor',
-				'wp-nux',
+				'wp-element',
 				'wp-plugins',
-				'wp-polyfill',
```

Comment:
Don't we need `wp-polyfill` for IE11 support?

Reply:
It's already handled by `Jetpack_Gutenberg`:

https://github.com/Automattic/jetpack/blob/0b8ffe1615f2b8bc09e22e3a5f065f7674a65fdc/class.jetpack-gutenberg.php#L596